resolwe/elastic/indices.py

```diff
@@ -0,0 +1,122 @@
+"""ElasticSearch indices."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import operator
+import six
+
+import elasticsearch_dsl as dsl
+
+from guardian.models import GroupObjectPermission, UserObjectPermission
+
+from resolwe.flow.models import dict_dot
+
+
+class NoValue(object):
```

Comment:
Please explain that this is used to differentiate missing attributes from `None` (another approach would be to catch `AttributeError` instead of having a custom guard type).